modules/sonobiBidAdapter.js

```diff
@@ -56,15 +56,23 @@ export const spec = {
    */
   buildRequests: (validBidRequests, bidderRequest) => {
     const bids = validBidRequests.map(bid => {
+      let mediaType;
+      if (deepAccess(bid, 'mediaTypes.banner')) {
+        mediaType = 'display';
+      }
+      if (deepAccess(bid, 'mediaTypes.video')) {
+        mediaType = 'video';
+      }
```

Comment:
Can this be written as the following so this does not run both conditions if it is not necessary:

```
if (deepAccess(bid, 'mediaTypes.video')) {
        mediaType = 'video';
} else if (deepAccess(bid, 'mediaTypes.banner')) {
        mediaType = 'display';
}      
```